src/tcpedit/plugins/dlt_linuxsll2/Makefile.am

```diff
@@ -0,0 +1,30 @@
+# $Id:$
+# START OF: dlt_linuxsll2
```

Comment:
Need copyright stuff here

Reply:
Left out this one, since there's no `Makefile.am` to copy+paste Copyright stuff from.